python/tests/test_detail.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Should we mimick module structure and put this into `tests/detail/`?

Reply:
Should I also move the `test_neutron.py` and `test_plot.py` into their own directories?

Reply:
I believe Dan just moved the former in his PR.